package.json

```diff
@@ -15,7 +15,8 @@
     "prettier": "prettier --write '**/*.{js,json,css,md}'",
     "test": "flow check",
     "flow": "flow",
-    "lint": "eslint --ext .js ."
+    "lint": "eslint --ext .js .",
+    "reset": "rm -Rf dist/ node_modules dist/node_modules && yarn && cd dist && yarn"
```

Comment:
🤔 

I'm very much not sure about this– I'd figured a "reset" would do something like: 

- Remove `dist` directory
- Remove root `node_modules` directory
- Remove demo `node_modules` directory
- Run `yarn` in root
- Run `yarn` in demo

Don't think our `dist` directory ever has a `node_modules` directory inside of it.

Also `rm -Rf` inside of a package.json script scares me. 🙈